infra/resources/_modules/cosmos/io_com_cosmos.tf

```diff
@@ -0,0 +1,73 @@
+module "io_com_cosmos_account" {
+  source = "github.com/pagopa/dx//infra/modules/azure_cosmos_account?ref=79b311a72bed34867e6ea9dff9495a44b5419afc"
+
+  subnet_pep_id = var.subnet_pep_id
+
+  alerts = {
+    enabled         = true
+    action_group_id = var.action_group_id
+    thresholds = {
+      provisioned_throughput_exceeded = 1000
+    }
+  }
+
+  environment = var.environment
+
+  tags = var.tags
+
+  consistency_policy = {
+    consistency_preset = "Default"
+  }
+
+  resource_group_name = var.resource_group
+}
+
+module "cosmosdb_sql_database_data_lake" {
```

Comment:
I suggest to replace all these dependencies (`cosmosdb_sql_database`) with the actual resource. Code is identical, but modules are going to block any update on this repo very soon:

```hcl
resource "azurerm_cosmosdb_sql_database" "data_lake" {
  name                = "data-lake"
  resource_group_name = var.resource_group
  account_name        = module.io_com_cosmos_account.name
}
```

Reply:
@Krusty93 Ok thanks for the suggestion, I made a small refactor here 😄 [a3a69bd](https://github.com/pagopa/io-messages/pull/88/commits/a3a69bdc1a515421254e02e27fddaece125e8e3f)